webapp/src/pages/home/Home.tsx

```diff
@@ -197,6 +198,21 @@ function Home() {
         },
     };
 
+    // @ts-expect-error _ and __ are unused
+    const onClickTreemapEntry = useCallback(async (_, __, config) => {
+        if (db && author !== defaultSelection) {
+            const repo = config.globals.categoryLabels[config.dataPointIndex];
+            const fullRepos = await runQuery(db, `SELECT owner, name FROM main_marts.fct_repositories WHERE name='${repo}';`);
+            const fullJsonRepos = fullRepos.toArray().map(i => i.toJSON());
+            // in unique scenarios, such as with a lot of forks from one repository
+            // there are cases when one author has PRs on multiple repo owners for the same repo name
+            fullJsonRepos.forEach(fullRepo => {
+                const url = `https://github.com/${fullRepo?.owner}/${fullRepo?.name}/pulls?q=is:pr+author:${author}`;
```

Comment:
@ramonvermeulen : We could move this url into a column set within the plugin as this url is originally defined [there](https://github.com/godatadriven/github-contributions/blob/ffd8b5db9c61bcb96225849cd07d6089c50e9695/github_contributions/src/github_contributions/api.py#L164)